mongo-movies/app.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const path = require("path");
+const favicon = require("serve-favicon");
+const logger = require("morgan");
+const cookieParser = require("cookie-parser");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+
+const index = require("./routes/index");
+const celebrity = require("./routes/celebrity");
+
+mongoose.connect("mongodb://localhost/movies-dev", {
+  keepAlive: true,
+  reconnectTries: Number.MAX_VALUE,
+  useMongoClient: true
+});
+
+const app = express();
+
+// view engine setup
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "ejs");
+
+// uncomment after placing your favicon in /public
+//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+app.use(logger("dev"));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, "public")));
+
+app.use("/", index);
+app.use("/celebrity", celebrity);
+// app.use("/celebrity/show", celebrity);
+app.use("/celebrity/new", celebrity);
```

Comment:
You don't need to use this route (neither the ones on lines 36 and 37). It's enough with the one in line 33.